build.gradle

```diff
@@ -1226,15 +1226,27 @@ project(':generator') {
 }
 
 project(':clients') {
+  apply plugin: 'com.github.johnrengelman.shadow'
+
   archivesBaseName = "kafka-clients"
 
+  shadowJar {
+    zip64 = true
+    relocate('io.opentelemetry', 'shadow.io.opentelemetry')
+    relocate('com.google.protobuf', 'shadow.com.google.protobuf')
+  }
+
   dependencies {
     implementation libs.zstd
     implementation libs.lz4
     implementation libs.opentelemetryProto
     implementation libs.snappy
     implementation libs.slf4jApi
 
+    // kip-714 jar shadowing
+    implementation  'io.opentelemetry.proto:opentelemetry-proto:0.13.0-alpha'
+    implementation  'com.google.protobuf:protobuf-java:3.19.4'
```

Comment:
this is already included transitively as part of `libs.opentelemetryProto`. If we need to declare it explicitly we should use `lib.protobuf` which we already have.